### PR TITLE
Add panel for vacancies with some placeholder text.

### PR DESCRIPTION
**Pull request by ian-r-rose**
Partial fix for #77

### CODE REVIEW

File: public/js/Sidebar.js [hunk 1]
```diff
@@ -25,16 +25,49 @@ var app = this.app || {};
     this.address                = document.getElementById('sidebar-address');
     this.streetSegment          = document.getElementById('sidebar-street-segment');
     this.closeButton            = document.getElementById('sidebar-close-button');
+
+    // Vacant panel elements
+    this.vacantContainer          = document.getElementById('sidebar-vacant');
+    this.vacantCommonName         = document.getElementById('sidebar-vacant-common-name');
+    this.vacantTreeId             = document.getElementById('sidebar-vacant-tree-id');
+    this.vacantAddress            = document.getElementById('sidebar-vacant-address');
+    this.vacantPruningYea  r      = document.getElementById('sidebar-vacant-pruning-year');
```
Comment (emillipede) [inline public/js/Sidebar.js]: this.vacantPruningYea  r  

this typo was causing a little problem

File: public/js/Sidebar.js [hunk 2]
```diff
@@ -66,6 +99,7 @@ var app = this.app || {};
 
   Sidebar.prototype.showDefault = function() {
     this.treeContainer.classList.add('hidden');
+    this.vacantContainer.classList.add('hidden');
```
Comment (emillipede) [inline public/js/Sidebar.js]: the hiddenness doesn't seem to be working quite right. once I click on a vacant marker, the vacant sidebar never disappears. the vacant sidebar also seems permanently stuck under the tree sidebar. 

vacant marker selected:
Screen Shot 2019-07-22 at 9 55 53 PM

tree marker selected:
Screen Shot 2019-07-22 at 9 52 04 PM

Comment (ian-r-rose) [reply public/js/Sidebar.js]: Good catch, should be fixed now.

**Comment by ian-r-rose** [comment]
Thanks for the review!

**Comment by emillipede** [comment]
for future reference: the close buttons in each panel are independently formatted

**Pull request merged**